### PR TITLE
Feature 177 various style improvements

### LINKED ISSUE #177: Various style improvements
- Background image heading multiline fix
- Breadcrumbs
- Application settings
- etc.

### PR DESCRIPTION
**Pull request by Kaemppi**
resolves #177

### CODE REVIEW

File: src/styles/editor-view.ts [hunk 1]
```diff
@@ -15,6 +15,31 @@ export default createStyles({
     overflowY: "auto",
   },
 
+  treeLoaderContainer: {
+    top: 54,
+    left: 0,
+    right: 0,
+    paddingTop: theme.spacing(4),
+    position: "absolute",
```
Comment (anttileppa) [inline src/styles/editor-view.ts:23]: Should this be fixed to prevent issues with scrolling?

File: src/styles/editor-view.ts [hunk 2]
```diff
@@ -15,6 +15,31 @@ export default createStyles({
...
+    alignItems: "center",
+    background: "#fff"
+  },
+
+  loaderContainer: {
+    top: 0,
+    left: 300,
+    right: 0,
+    bottom: 0,
+    position: "absolute",
```
Comment (anttileppa) [inline src/styles/editor-view.ts:36]: Should this be fixed to prevent issues with scrolling?

**Review by anttileppa — CHANGES_REQUESTED** [review]

File: src/styles/editor-view.ts (hunk 1 above)
Comment (Kaemppi) [reply src/styles/editor-view.ts:23]: No, it is positioned within parent element, not the whole view.

File: src/styles/editor-view.ts (hunk 2 above)
Comment (Kaemppi) [reply src/styles/editor-view.ts:36]: No, it is positioned within parent element, not the whole view.

File: src/styles/editor-view.ts (hunk 1 above)
Comment (anttileppa) [reply src/styles/editor-view.ts:23]: ok

File: src/styles/editor-view.ts (hunk 2 above)
Comment (anttileppa) [reply src/styles/editor-view.ts:36]: ok

**Pull request merged**